client/stylesheets/_responsive.scss

```diff
@@ -244,15 +350,18 @@
     // .homeCardLi {
     // 	width: 40%;
     // }
+    .searchResultsCardImg {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardImg` should be written in lowercase with hyphens